### ISSUE TITLE
Invalid command line     Unknown option phyi

### ISSUE THREAD

**Issue by tcd11221** [thlee/SNPhylo#52]
The following error was encountered when using the software, the chromosome has been changed to a number and the CTG has been removed：

Hint: it is suggested to call `snpgdsOpen' to open a SNP GDS file instead of `openfn.gds'.
SNP pruning based on LD:
Excluding 0 SNP on non-autosomes
Excluding 226,074 SNPs (monomorphic: TRUE, MAF: 0.1, missing rate: 0.1)
    # of samples: 31
    # of SNPs: 523,834
    using 1 thread
    sliding window: 500,000 basepairs, Inf SNPs
    |LD| threshold: 0.1
    method: composite
Chromosome 1: 0.21%, 256/121,027
Chromosome 2: 0.24%, 261/109,355
Chromosome 3: 0.21%, 252/121,945
Chromosome 4: 0.19%, 246/126,613
Chromosome 5: 0.24%, 202/85,247
Chromosome 6: 0.20%, 257/127,325
Chromosome 7: 0.20%, 118/58,396
1,592 markers are selected in total.
Warning: The length of sequence is too short (< 2000 bp) to construct a good tree!
Please consider to restart this script with different parameter values (-l, -m and/or -M).

Invalid command line
Unknown option phyi

I am very anxious and need help, thanks

**Comment by MonaLiu421** [comment]
Hi, have you solved this problem? I also encountered this.

**Comment by tcd11221** [comment]
> Hi, have you solved this problem? I also encountered this.

YES, you can check the format of the input vcf file, note that chromosome names can only be numbers, no characters like Chr etc.

**Comment by MonaLiu421** [comment]
Thanks for your reply. The chromosome names in my vcf file are numbers.

**Comment by MonaLiu421** [comment]
![image](https://github.com/thlee/SNPhylo/assets/45484925/f6bf9fff-6b58-49ed-b1f9-380608eb3790)

**Comment by SureINRA** [comment]
> > Hi, have you solved this problem? I also encountered this.
> 
> YES, you can check the format of the input vcf file, note that chromosome names can only be numbers, no characters like Chr etc.

Hi tcd11221, I face the same trouble as MonaLiu421, chrom=numbers, ID is not ".", as blew:

###
Hint: it is suggested to call `snpgdsOpen' to open a SNP GDS file instead of `openfn.gds'.
SNP pruning based on LD:
Excluding 0 SNP on non-autosomes
Excluding 7,055,166 SNPs (monomorphic: TRUE, MAF: 0.1, missing rate: 0.1)
    # of samples: 319
    # of SNPs: 1,480,875
    using 1 thread
    sliding window: 500,000 basepairs, Inf SNPs
    |LD| threshold: 0.1
    method: composite
Chromosome 1: 0.04%, 865/1,952,553
Chromosome 2: 0.06%, 631/984,820
Chromosome 3: 0.06%, 543/973,457
Chromosome 4: 0.06%, 569/1,004,516
Chromosome 5: 0.04%, 346/786,651
Chromosome 6: 0.05%, 573/1,162,964
Chromosome 7: 0.05%, 456/880,451
Chromosome 8: 0.06%, 450/790,629
4,433 markers are selected in total.


Invalid command line
Unknown option phyi

finished
###

I am still no idea to solve this problem.

Shuo LIU

**Comment by plant-bioinfo** [comment]
I apologize for the inconvenience.

The issue arose because the latest version of the muscle program no longer supports the specified option (-phyi).
To resolve this error, you can either use an older version of muscle (https://drive5.com/muscle/downloads_v3.htm) or run snphylo without the '-A' option.

Thank you.

Tae-Ho Lee

**Comment by SureINRA** [comment]
Hi Tae-Ho Lee,
thanks a lot, It works if I cancel the -A option.
Shuo
